code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -807,7 +805,7 @@
 /datum/reagent/consumable/caramel
 	name = "Caramel"
 	description = "Who would have guessed that heated sugar could be so delicious?"
-	nutriment_factor = 10 * REAGENTS_METABOLISM
+	nutriment_factor = 4 * REAGENTS_METABOLISM
```

Comment:
this is also actual food (ingame)

Reply:
Its just sweetness and a lighter to make a flat out better food chem.